src/shogun/evaluation/ClusteringAccuracy.cpp

```diff
@@ -23,6 +23,9 @@ float64_t CClusteringAccuracy::evaluate(CLabels* predicted, CLabels* ground_trut
 	int32_t correct=0;
 	for (int32_t i=0; i < predicted_ilabels.vlen; ++i)
 	{
+		/* print progress */
+		SG_PROGRESS(i, 0, predicted_ilabels.vlen, 1, "EVALUATING ACCURACY: ");
```

Comment:
Yeah I mean like accuracy is just an average of a vector difference so there is no loop needed for doing that but we can just use a linalg or eigen3 call which will SIMD it